web/src/components/ui/select.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react'
+
+import {
+  CaretSortIcon,
+  CheckIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+} from '@radix-ui/react-icons'
+import * as SelectPrimitive from '@radix-ui/react-select'
+
+import { cn } from 'src/lib/utils'
+
+const Select = SelectPrimitive.Root
+
+const SelectGroup = SelectPrimitive.Group
+
+const SelectValue = SelectPrimitive.Value
+
+const SelectTrigger = React.forwardRef<
+  React.ElementRef<typeof SelectPrimitive.Trigger>,
+  React.ComponentPropsWithoutRef<typeof SelectPrimitive.Trigger>
+>(({ className, children, ...props }, ref) => (
+  <SelectPrimitive.Trigger
```

Comment:
I got a strange situation where focusing on the button will change the border radius of the button to more rounded (kind of full?). Not critical. But I just wanted to check in to see if you have any ideas on why it's happening. 🤞

![image](https://github.com/user-attachments/assets/8b6597e6-12bb-4bfa-9c92-abfeffa8b596)